test/create_envs.sh

```diff
@@ -0,0 +1,10 @@
+echo "VOLUMES=$VOLUMES" >> .env
+echo "DATASETS=$DATASETS" >> .env
+echo "EXPERIMENTER=$EXPERIMENTER" >> .env
+echo "DEFAULT_MONGO_PORT=$DEFAULT_MONGO_PORT" >> .env
+echo "DOCKER_HOSTNAME=$DOCKER_HOSTNAME" >> .env
+echo "REAL_MONGO_PORT=$REAL_MONGO_PORT" >> .env
+echo "REDIS_PORT=$REDIS_PORT" >> .env
+echo "REDIS_HOST=$REDIT_HOST" >> .env
+echo "REAL_MONGO_PORT_DEV=$REAL_MONGO_PORT_DEV" >> .env
+echo "MODE=$MODE" >> .env
```

Comment:
I'm curious @orionw, is this `.env` file generation script needed, since all the env variables injected by it are already injected? (At least it seems like they're already injected since we're using them here to generate a `.env` file?) I could be way off.

Reply:
So yes and no - the env files will already be present in the Travis-CI but the easiest way to pass all of those envs to docker is through a .env file.  Otherwise it seems like you have to list them in the docker-compose file (which might be cleaner actually now that I thjink about it? I'm open to switching to that if you agree that it'd be cleaner).  https://docs.docker.com/compose/environment-variables/

Reply:
I think the way you've got it is just fine for now; my question was a small potato question.